lib/metadata/ModelVersion.md

```diff
@@ -47,10 +47,19 @@ Used to store the server bucket encryption info
 
 ### Properties Added
 
-```
-this._websiteConfiguration = websiteConfiguration || null;
+```javascript
+this._locationConstraint = LocationConstraint || null;
 ```
 
 ### Usage
 
-Used to store the bucket website configuration info
+Used to store the bucket website configuration info and the location constraint
```

Comment:
@bennettbuchanan I went ahead and made this change
Thanks for spotting it @LaurenSpiegel